test/parallel/test-performance-measure-detail.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+require('../common');
+const assert = require('assert');
+const util = require('util');
+const { performance, PerformanceObserver } = require('perf_hooks');
+
+const perfObserver = new PerformanceObserver((items) => {
+  items.getEntries().forEach((entry) => {
+    assert.ok(util.inspect(entry).includes('this is detail'));
+  });
```

Comment:
Should this be wrapped with `common.mustCall`?

Reply:
Agree 👍

Reply:
Done ! Thanks !